mlrun/model_monitoring/applications/base.py

```diff
@@ -271,7 +272,12 @@ def _handler(
                 "working with endpoints, without any custom data-frame input"
             )
 
-        application_name = self.__class__.__name__
+        if not application_name:
+            application_name = self.__class__.__name__
```

Comment:
I think we should normalize it and to make sure the user want get any warning.
```suggestion
            application_name = mlrun.utils.normalize_name(self.__class__.__name__, False)
```

Reply:
Addressed - check via the regex pattern